code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -40,6 +40,16 @@
 	if(W.tool_behaviour == TOOL_WRENCH && !(flags_1&NODECONSTRUCT_1))
 		W.play_tool_sound(src)
 		deconstruct(TRUE)
+	else if(istype(W, /obj/item/bedsheet))
+		if(user.transferItemToLoc(W, drop_location()))
+			to_chat(user, "<span class='notice'>You make \the [src] with [W].</span>")
+			W.pixel_x = 0
+			W.pixel_y = 0
+	else if(istype(W, /obj/item/disk/nuclear))
```

Comment:
As someone pointed out in another repository, plushies should also be able to be put to bed.
Perhaps you could make this behaviour into an element that can be applied to both items.

Reply:
yeah i'll do a tuckable element